TwilioVerifyDemo/TwilioVerifyDemo/CreateFactor/Presenter/CreateFactorPresenter.swift

```diff
@@ -43,6 +43,23 @@ class CreateFactorPresenter {
     self.twilioVerify = twilioVerify
     self.accessTokensAPI = accessTokensAPI
   }
+  
+  /// This method is intended to demonstrate how to access to
+  /// the server custom errors, such custom response codes and details.
+  /// - Parameter error: Create Factor result error
+  private func retrieveServerErrorDetail(
+    with error: TwilioVerifyError
+  ) {
+    switch error.originalError as? NetworkError {
+      case .failureStatusCode(let response):
+        // Gets server response code
+        var errorResponse = response
+        print("Custom server response: \(errorResponse.serverResponse as Any)")
```

Comment:
Can you add the defined dialog when the error is 60315?

Reply:
Showing the message + code 

![image](https://user-images.githubusercontent.com/14830877/148406049-5391725a-5c47-4b73-bf9b-c548074059b5.png)